test/resources/examples/index.js

```diff
@@ -262,6 +262,51 @@ const shortRingBetterTrade = generateTestCase({
   ],
 })
 
+const maxUint128 = new BN(2).pow(new BN(128)).sub(new BN(1))
+const exampleNoUsedAmountTracking = generateTestCase(
+  {
+    deposits: [
+      { amount: feeAdded(toETH(5)), token: 0, user: 0 },
+      { amount: feeAdded(toETH(5)), token: 1, user: 1 },
+    ],
+    orders: [
+      {
+        sellToken: 0,
+        buyToken: 1,
+        sellAmount: maxUint128,
+        buyAmount: feeSubtracted(maxUint128.div(new BN(4))),
```

Comment:
Why do we need 4 orders here? Would it make sense to have two orders, one where the buyAmount is unlimited and one where the sellAmount is unlimited?

Reply:
I do not want to reuse the first order in the second batch submission, in order to check that the reversion has not updated the usedAmount. The easiest example, which I could come up with was exactly this 4 order example.

Yeah, I could also set the buyAmount to maxUint128 in one case. I left it out originally because it might make the test case more complicated. But yeah, maybe it is safer if I do it.